stone/target/python_rsrc/stone_serializers.py

```diff
@@ -232,6 +256,8 @@ def encode_primitive(self, validator, value):
             return None
         elif isinstance(validator, bv.Timestamp):
             return _strftime(value, validator.format)
+        elif isinstance(validator, bv.Iso8601Timestamp):
+            return _strftime(value.astimezone(TZINFO_UTC), validator.output_format)
```

Comment:
This is to support the case where the `datetime` object we're provided is in a different timezone. This basically imposes that all encoded timestamps are always in UTC. This is basically so format strings with hard-coded timezone values (e.g., `%Y-%m-%dT%H:%M:%SZ`, `%Y-%m-%dT%H:%M:%S +0000`) will work. You can still trick it by lying (e.g., `%Y-%m-%dT%H:%M:%S +1234`), but the former (anti)pattern is common enough where I think it makes sense.